apps/desktop/src-tauri/src/backend.rs

```diff
@@ -82,6 +82,10 @@ pub fn initialize_rudder_analytics(key: String, data_plane: String) {
             true => Some(event),
             false => None,
         })),
+        package_metadata: Some(analytics::PackageMetadata {
+            name: env!("CARGO_CRATE_NAME"),
+            version: env!("CARGO_PKG_VERSION"),
```

Comment:
We should probably use a git hash here instead of cargo_pkg version. https://crates.io/crates/git-version